packages/layout/src/Box/Box.constants.ts

```diff
@@ -0,0 +1,62 @@
+export const VARIANT = {
+  ghost: "bg-transparent",
+};
+
+export const BOX_PROPS = {
+  p: {
+    rule: "padding",
+    type: "spacing",
+  },
+  m: {
+    rule: "margin",
+    type: "spacing",
+  },
```

Comment:
Notice that this is now an object. This makes it so we can access each box prop by object key instead of needing to loop through this each time.